### DISCUSSION TITLE
Get BCNA page

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#76] [Ideas]
This page presents various options, two of which lead to a pool page. For new users, it may be unclear what steps to take from there in order to purchase BCNA tokens.

![Screenshot 2023-05-24 at 19 44 03](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/e7c5578e-68b9-432b-b56c-3aa69a1e3308)

What should I do from here?

![Screenshot 2023-05-24 at 21 42 42](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/cdef0d3d-0b0c-44b8-9e00-2f3212d5383d)

Not better to redirect to : https://app.osmosis.zone/?from=USDC&to=BCNA for example?

**Comment by TJ-BitCanna** [comment]
Yes! Very good points!

We'll take this with us!

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- replace the 2 Osmosis pools with 1 box linking directly to https://app.osmosis.zone/?from=USDC&to=BCNA

**Comment by TJ-BitCanna** [comment]
Tested and approved.
